src/core/assets/KavaAILogo.tsx

```diff
@@ -1,11 +1,23 @@
-const KavaAILogo = () => {
+interface KavaAILogoProps {
+  width?: number;
+  height?: number;
+}
+
+const KavaAILogo: React.FC<KavaAILogoProps> = ({ width, height }) => {
```

Comment:
Made the KavaAI logo size adjustable so it can be used in different parts of the app. Also made sure the aspect ratio stays the same if we only want to provide a height or a width during usage